setup-r-dependencies/README.md

```diff
@@ -14,7 +14,7 @@ This action install dependencies for the current R environment based on the DESC
 Inputs available
 
 - `cache` - default `true`. Whether packages should be cached across runs or
```

Comment:
```suggestion
- `cache` - default `"always"`. Whether packages should be cached across runs or
```

Reply:
I'll remove the pruning from this PR, so let's keep this opt-in.